.travis.yml

```diff
@@ -2,6 +2,5 @@ language: ruby
 cache: bundler
 sudo: false
 rvm:
-  - 2.0.0
-  - 2.1
-  - 2.2
+  - 2.1.8
```

Comment:
Safe to drop 2.1.x and keep somewhat in line with https://github.com/test-kitchen/test-kitchen/blob/master/.travis.yml#L3-L6